lib/create_api_gem/forms/settings/settings.rb

```diff
@@ -86,6 +93,6 @@ def self.full_example(email_block_for_notifications_ref)
     image = { href: APIConfig.image_api_request_url + '/images/default' }
     Settings.new(redirect_after_submit_url: 'http://google.com', show_typeform_branding: false, progress_bar: 'percentage',
                  show_progress_bar: false, description: 'some meta description', allow_indexing: true, image: image,
-                 language: 'fr', is_public: true, google_analytics: 'UA-1234-12', notifications: Notifications.full_example(email_block_for_notifications_ref))
+                 language: 'fr', is_public: true, google_analytics: 'UA-1234-12', facebook_pixel: '12345678901234567', notifications: Notifications.full_example(email_block_for_notifications_ref))
```

Comment:
This line is good & needed but is also why the tests are failing, they are hitting production so until this is implemented they will fail